pkgs/build-support/fetchsteam/depotdownloader_2_5_0/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+}:
+
+buildDotnetModule rec {
+  pname = "depotdownloader";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "SteamRE";
+    repo = "DepotDownloader";
+    rev = "DepotDownloader_${version}";
+    sha256 = "Kgi0u+H5BIAhrjk9e+8H1h0p5Edm3+2twYBPY3JQGps=";
+  };
+
+  projectFile = "DepotDownloader.sln";
+  nugetDeps = ./deps.nix;
+
```

```suggestion

  postFixup = ''
    makeBinaryWrapper $out/bin/DepotDownloader \
      --set-default SSL_CERT_FILE "${cacert}/etc/ssl/certs/ca-bundle.crt"
  '';

```
This could help us avoid setting it in the fetcher